community/kernel/src/main/java/org/neo4j/graphdb/factory/GraphDatabaseSettings.java

```diff
@@ -115,7 +115,7 @@
     @Internal
     public static final Setting<String> cypher_runtime = setting(
             "dbms.cypher.runtime",
-            options( "INTERPRETED", "COMPILED", DEFAULT ), DEFAULT );
+            options( "INTERPRETED", DEFAULT ), DEFAULT );
```

Comment:
This is now a very useful config option :p

It's fine leaving it in favour of less change, of course.